src/components/pages/PageCreateCards/PageCreateCards.jsx

```diff
@@ -0,0 +1,140 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import TemplateDefault from '../../Templates/TemplateDefault';
+import firebase from '../../../firebase/firebase';
+
+function PageCreateCards() {
+  const [cardName, setCardName] = React.useState('');
```

Comment:
I don't think you need 6 different useState values here. You should be able to have one card state value instead `const [card, setCard] = useState({})`.